SIDFactoryII/main.cpp

```diff
@@ -25,14 +26,29 @@ void BuildResource();
 // Functions
 int main(int inArgc, char* inArgv[])
 {
-	//BuildResource();
-    
+#ifdef _BUILD_NR
+	const char* build_number = _BUILD_NR;
+#else
+	const char* build_number = __DATE__;
+#endif
+
+#ifdef _SF2_WINDOWS
+	const char* os = "Windows";
+#else
+#ifdef _SF2_LINUX
+	const char* os = "Linux";
+#else
+	const char* os = "macOS";
+#endif
+#endif
+
+	Utility::Logging::instance().Info("SIDFactoryII %s build %s", os, build_number);
+
 	// Initialize SDL
 	const int sdl_init_result = SDL_Init(SDL_INIT_TIMER | SDL_INIT_AUDIO | SDL_INIT_VIDEO);
 	if (sdl_init_result < 0)
 	{
-		std::cout << "SDL initialization failed. SDL Error: " << SDL_GetError();
-
+		std::cout << "SDL initialization failed. SDL Error:" << SDL_GetError();
```

Comment:
The extra space was there for formatting the ouput so that there's a space after the colon.

Reply:
My mistake, I meant to put back the original code after I changed this to use the logging singleton. Didn't think like a good idea to use SDL logging if SDL couldn't be initialized.

Reply:
Fixed.